notebooks/02-quality-flags/TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,1011 @@
+{
```

Comment:
<p>I think it's also worth pointing out some "suspicious" characteristics about this folded light curve at this time, so readers can be on the lookout for them in the future. The two I see are that some of the values are 0, which would mean a planet was as big or larger than its sun. Actually all of the values are way too small for reasonable sized planet/star ratio. And, there is no clear "down and up" pattern when folded, the low dots seem randomly distributed. (Maybe that's 3 things...)</p><p> </p><p>Oh also - is this folding at a phase of like 1.25 days? That looks to be the max power form the periodogram. But I feel like the "naive by eye" look at the original light curve would indicate a period of the 2.5 days. Hence why there appear to be two dips, at 0 and half an orbital phase. We could show both? Or it could be an in-line exercise, something like: "The period was at max power at about 1.25 days, but by eye the dips appear every 2.5 days. What does the folded light curve look like if you use the 2nd-highest max power period from the periodogram?"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>will write markdown cell pointing out the behavior you mention here. I did try to fold it on many different periods, ~2.5 days, the exact 2nd maximum pariod, 2x and 4x the maximum period and they all looked like this. I will add more info but I'm not sure how to make it look more convincing </p><p> </p>